### PR TITLE
Sudivate/scoringimage

### PR DESCRIPTION
**Pull request by sudivate**
**Demonstrate model deployment to Azure Web App for containers**  

- Correcting the spelling on the doc 
- Adding scripts to create an image using AML SDK
- Update getting started guide to include deployment to Web App

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -190,25 +190,48 @@ In a similar way create a stage **Prod (AKS** and add a single task to the job *
...
+| Deployment Name                   | mlopspython-aks                                                                                      |
+| Deployment Configuration file     | `$(System.DefaultWorkingDirectory)/_ci-build/mlops-pipelines/code/scoring/deployment_config_aks.yml` |
+| Overwrite existing deployment     | X                                                                                                    |
 
 **Note:** Creating of a Kubernetes cluster on AKS is out of scope of this tutorial, so you should take care of it on your own.
 
-Save the pipeline and craete a release to trigger it manually. Once the pipeline exection is finished, check out deployments in the **mlops-AML-WS** workspace.
+**Deploy trained model to Azure Web App for containers**
+
+[Create Image Script](./ml_service/util/create_scoring_image.py)
```
Comment (eedorenko) [inline docs/getting_started.md]: The link doesn't work

File: docs/getting_started.md [hunk 2]
```diff
@@ -190,25 +190,48 @@ In a similar way create a stage **Prod (AKS** and add a single task to the job *
...
+can be used to create a scoring image from the release pipeline. Image created by this script will be registered under Azure Container Registry(ACR) instance that belongs to Azure Machine Learning Service. Any dependencies that scoring file depends on can also be packaged with the container with Image config. To learn more on how to create a container with AML SDK click [here](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.image.image.image?view=azure-ml-py#create-workspace--name--models--image-config-).    
+
+Below is release pipeline with two tasks one to create an image using the above script and second is the deploy the image to Web App for containers  
+![release_webapp](./images/release-webapp-pipeline.PNG)
+
+Using bash script task to invoke [Create Image Script](./ml_service/util/create_scoring_image.py)
```
Comment (eedorenko) [inline docs/getting_started.md]: The link doesn't work

File: docs/getting_started.md [hunk 3]
```diff
@@ -190,25 +190,48 @@ In a similar way create a stage **Prod (AKS** and add a single task to the job *
...
+| Model Deployment Target           | Azure Kubernetes Service                                                                             |
+| Select AKS Cluster for Deployment | YOUR_DEPLOYMENT_K8S_CLUSTER                                                                          |
+| Deployment Name                   | mlopspython-aks                                                                                      |
+| Deployment Configuration file     | `$(System.DefaultWorkingDirectory)/_ci-build/mlops-pipelines/code/scoring/deployment_config_aks.yml` |
+| Overwrite existing deployment     | X                                                                                                    |
 
 **Note:** Creating of a Kubernetes cluster on AKS is out of scope of this tutorial, so you should take care of it on your own.
 
-Save the pipeline and craete a release to trigger it manually. Once the pipeline exection is finished, check out deployments in the **mlops-AML-WS** workspace.
+**Deploy trained model to Azure Web App for containers**
```
Comment (eedorenko) [inline docs/getting_started.md]: It's not clear why we should do that. Perhaps it is worth mentioning, that it is an optional step if you don't work with ACI or AKS and prefer Web App

File: docs/getting_started.md (hunk 1 above)
Comment (sudivate) [reply docs/getting_started.md]: fixed

**Pull request merged**